tests/installation/install_service.pm

```diff
@@ -16,7 +16,12 @@ use main_common 'is_desktop';
 
 sub run {
 
-    select_console 'root-console';
+    if (get_var('SEL_SERIAL_CONSOLE')) {
```

Comment:
I am not like introduce new setting, can hpc directly use serial console?

Reply:
Although this requirement comes from HPC, but I do want to select serial console for whole service check through the switch of 'SEL_SERIAL_CONSOLE' when needed.